### PR TITLE
[Fix] Properly style single urls in a bigger text

### PR DESCRIPTION
**Pull request by SferaDev**
- We styled the whole content if a single link was found
- Add ``react-linkify`` as we did in the past (see vaccination-app)

Before:

image

After:

image

### CODE REVIEW

**Review by adrianq — APPROVED** [review]
@tokland This works as expected. Will wait for your review before merging it

**Review by tokland — APPROVED** [review]
Tested with project-monitoring-app, all good :+1:

**Pull request merged**
